docs/content/usage/configuration/credentials.md

```diff
@@ -5,6 +5,27 @@ draft: false
 description: "Configure credentials that the Infection Monkey will use for propagation."
 ---
 
-On this screen you can feed the Infection Monkey "stolen" credentials from your network, simulating an attacker with inside knowledge.
+On this screen, you can configure credentials for Infection Monkey to use during propagation.
```

Comment:
I don't like "during propagation". If only vulnerability plugins are enabled the credentials won't be used. I think stating that brute-force plugins use these credentials is both: more accurate and easier to understand.